packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md

```diff
@@ -6,6 +6,148 @@ section: get-started
+    - [@patternfly/react-virtualized-extension@6.1.0](https://www.npmjs.com/package/@patternfly/react-virtualized-extension/v/6.1.0) ([changelog](https://github.com/patternfly/react-virtualized-extension/releases/tag/v6.1.0))
+
+### React 19 support
+
+PatternFly now includes support for React 19 across all components and extensions, enabling you to benefit from  new React features and performance enhancements.
+
+This update is non-breaking, and PatternFly 6 will continue to support React 17 and 18 in line with our support policy. In future releases, we plan to more fully leverage React 19's capabilities to enhance existing PatternFly features and introduce new ones.
+
+### Component animations
+
+As we progress in our effort to integrate motion design across PatternFly components, we added the following new animations in this release. To provide a holistic view of our animated components, we also put together [an interactive animations demo](/design-foundations/motion/react-demos), which we will continue to update as we add additional animation support.   
+
+We have also created an [enable-animations codemod](https://github.com/patternfly/pf-codemods/tree/main/packages/eslint-plugin-pf-codemods/src/rules/v6/enableAnimations), which adds the `hasAnimations` prop to components that support opt-in animations.  
+
+#### Expansion 
+
+We added expansion animations that apply a fade-in and slide-in transition as hidden content sections are expanded in a variety of components. A similar, reversed animation applies when expanded sections are collapsed.
+
+The following list highlights opt-in requirements and links to examples. Unless an animation is marked with "Requires opt-in," it will be enabled by default. 
+
+- Accordion 
+    - **Example:** All [accordions](/components/accordion) in our examples and demos.
+
+- Dual list selector 
+    - **Requires opt-in.**
+    - **Example:** [Dual list selector with tree](/components/dual-list-selector#with-tree), which has opted in.
+
+- Expandable section 
+    - **Example:** All [expandable sections](/components/expandable-section) in our examples and demos.
+
+- Form
+    - **Requires opt-in.**
+    - **Example:** [Form field groups](/components/forms/form#field-groups), which have opted in.  
+
+- Navigation
+    - **Example:** [Expandable navigation examples](/components/navigation#expandable).
+    - **Note:** This animation was introduced in our 6.2 release, but performance issues were reported. For this release, we reworked the animation to resolve these issues.
+
+- Search input   
+    - **Example:** [Expandable search input](/components/search-input#with-expandable-button).
+
+- Tree view
+    - **Requires opt-in.**
+    - **Example:** All [tree view](/components/tree-view) examples and demos, which have opted in. 
+
+- Table
+    - **Requires opt-in.**
+    - **Example:** [Expandable table](/components/table#expandable), which has opted in.
```

Comment:
Not sure if you wanted to call it out but ["compound expandable"](https://staging.patternfly.org/components/table#compound-expandable) is also animated (when you click on an open row item to close it, then click one to open it again - doesn't animate when you click between items in a row)